types.go

```diff
@@ -282,3 +283,22 @@ type BatchOptions struct {
 	ElemFlags uint64
 	Flags     uint64
 }
+
+//go:generate stringer -type LogLevel -trimprefix Log
+
+// LogLevel controls the verbosity of the kernel's eBPF program verifier.
+// These constants can be used for the ProgramOptions.LogLevel field.
+type LogLevel sys.LogLevel
```

Comment:
Stringer declares methods on `LogLevel`, which doesn't compile if LogLevel is an alias. Maybe I'm holding it wrong. I would also prefer an alias, because then we won't need the `LogLevel()` conversions anymore.

Reply:
Duh!

Reply:
After offline discussion, we'll skip the stringer and turn this into an alias for less tedium.